recipe_scrapers/madewithlau.py

```diff
@@ -56,7 +56,10 @@ def instructions(self):
         instructions = [
             p
             for pair in zip(headers, step_sets)
-            for p in [pair[0], *pair[1].find("div")]
+            for p in [
+                pair[0],
+                *(pair[1].find("div") if pair[1].find("div") else pair[1]),
```

Comment:
Possible simplification for this line (please check my logic on this):

```
*(pair[1].find("div") or pair[1]),
```

Reply:
Yeah, that looks much cleaner and works the same. Thanks!